t/unit/backends/test_base.py

```diff
@@ -80,9 +86,33 @@ def test_apply_chord(self, unlock='celery.chord_unlock'):
             uuid(),
             [self.app.AsyncResult(x) for x in range(3)],
         )
-        self.b.apply_chord(header_result, None)
+        self.b.apply_chord(header_result, self.callback.s())
```

Comment:
Why did you change this test?

Reply:
`fallback_chord_unlock` now uses the `body` argument instead of simply passing it on to `chord_unlock`, so where `None` worked before, it needs to be real signature now.